### PR TITLE
Landing page implementation Refactoring components to shadcn compoenents

### PR DESCRIPTION
**Pull request by xelleze**
This implementation is for incorporating shadcn components wherever it is possible for the existing code

### CODE REVIEW

File: frontend/app/globals.css
[the remark this answers is not in the document]
Comment (xelleze) [reply frontend/app/globals.css:1]: The changes where made automatically, but after the current review I added custom color and shadowbox to it

**Comment by Taza53** [comment]
@xelleze frontend/package-lock.json should be in gitignore and be deleted from the remote.

**Comment by xelleze** [comment]
@dIB59 ok so i tried to remove the file gave me errors about merge conflicts not sure if it is now correct can you check so I havent screwed something in the process of the merge from the dev branch

**Comment by dIB59** [comment]
Make sure to run npm run build before merging or pushing.
We do need package.json in the repo. We dont care about the lock.

**Comment by xelleze** [comment]
@dIB59 everything should be ok now I think

**Pull request merged**